### DISCUSSION TITLE
ImageViewer 相关的两个问题

### DISCUSSION THREAD

**Discussion by pisyell** [ant-design/ant-design-mobile#6366] [Q&A]
Version of antd-mobile
5.29.1

问题1：多图查看时候，当图片放大了为什么就不能进去左右切换了？能不能像微信一样可以支持在放大的时候也能切换图片，不然就得还原大小然后再切换。

问题2：滑动的时候slider位置不对，永远都多移动了 16 * n的距离

**Comment by 1587315093** [comment]
问题1 #6310

问题2是指放大后再滑动得位置吗？
